26-register_registration/resource_types/creator_selects_resource_type_report.feature

```diff
@@ -27,8 +29,10 @@ Feature: Creator selects Resource type Report
             | Working paper        |
             | Other type of report |
 
+    @test
     Scenario Outline: Creator sees that fields are validated for Resource subtypes for "Report"
         Given Creator sees fields for Resource subtypes for "Report"
+        And they have selected the Subtype "<Subtype>"
```

Comment:
Det er vel gitt i Given?
```suggestion
```

Reply:
Nei, hvis ikke "<Subtype>" er ned i scenarioet er det ingen måte å velge subtype på, derfor må nesten denne linjen være med. Alternativet er å endre `Scenario Outline` til `Scenario`, men da er det ingen måte å diferensiere mellom typene hvis vi har behov for det.

Reply:
sant, trengs for differensiering mellom typer ja